403.php

```diff
@@ -0,0 +1,27 @@
+<?php
+    include '_templates/sitewide.php';
+    $page['name'] = '403';
+    $page['title'] = 'Forbidden &sdot; elementary';
+    include $template['header'];
+?>
+
+<script>
+    ga('send', 'event', '403: Forbidden', window.location.host);
+</script>
+
+<div class="row">
+    <div class="column alert">
+        <i class="warning fa fa-warning"></i>
+    </div>
+    <div class="column alert">
+        <h3>Access denied.</h3>
+        <p>You don't have permission to view the requested resource.</p>
```

Comment:
Most of our in-page titles are longer and more friendly than this. Maybe change the paragraph to have a "Sorry, ..." at the beginning, since it's probably not that users fault.